packages/cli-e2e/utils/login.ts

```diff
@@ -0,0 +1,126 @@
+import {deletePassword, getPassword} from 'keytar';
+import {userInfo} from 'os';
+import type {Browser, Page, Target} from 'puppeteer-core';
+import type {ChildProcessWithoutNullStreams} from 'child_process';
+import {spawn} from 'child_process';
+import {answerPrompt, CLI_EXEC_PATH, isYesNoPrompt} from './cli';
+import LoginSelectors from './loginSelectors';
+import {isElementClickable} from './browser';
+
+function isLoginPage(page: Page) {
+  // TODO: CDX-98: URL should vary in fonction of the targeted environment.
+  return page.url() === 'https://platformdev.cloud.coveo.com/login';
+}
+
+export async function isLoggedin() {
+  const currentAccount = userInfo().username;
+  const accessToken = await getPassword(
+    'com.coveo.cli.access.token',
+    currentAccount
+  );
+  return accessToken !== null;
+}
+
+function waitForLoginPage(browser: Browser) {
+  return new Promise<Page>((resolve) => {
+    browser.on('targetchanged', async (target: Target) => {
+      const page = await target.page();
+      if (page && isLoginPage(page) !== null) {
+        resolve(page);
+      }
+    });
+  });
+}
+
+async function staySignedIn(page: Page) {
+  await page.waitForSelector(LoginSelectors.SubmitInput, {
+    visible: true,
+  });
+  await page.waitForSelector(LoginSelectors.SubmitInput);
+  await page.click(LoginSelectors.SubmitInput);
+}
+
+async function possiblyAcceptCustomerAgreement(page: Page) {
+  // TODO: CDX-98: URL should vary in fonction of the targeted environment.
+  if (page.url().startsWith('https://platformdev.cloud.coveo.com/eula')) {
+    await page.waitForSelector(LoginSelectors.coveoCheckboxButton);
+    await page.click(LoginSelectors.coveoCheckboxButton);
+    await page.waitForTimeout(200); // wait for the button to be enabled
```

Comment:
Can't you use isClickable here?

Reply:
I can't validate anymore since we already accepted the agreement with that account